app/src/main/java/org/mozilla/focus/activity/InfoActivity.java

```diff
@@ -104,7 +104,7 @@ public void onClick(View v) {
     @Override
     public View onCreateView(String name, Context context, AttributeSet attrs) {
         if (name.equals(IWebView.class.getName())) {
-            final View view = WebViewProvider.create(this, attrs, MainActivity.factory);
+            final View view = WebViewProvider.create(this, attrs, MainActivity.Companion.getFactory());
```

Comment:
nit: Add `@JvmStatic` to this method declaration and remove the additional `.Companion...`